### PR TITLE
feat(v2.3.x): support v2.3.x

### PR DESCRIPTION
**Pull request by XdpCs**
support v2.3.x

### CODE REVIEW

File: docs/en-US/v2.3.x/web/session/README.md
Comment (XdpCs) [inline docs/en-US/v2.3.x/web/session/README.md:1]: just review this.

File: docs/zh/v2.3.x/web/session/README.md
Comment (XdpCs) [inline docs/zh/v2.3.x/web/session/README.md:1]: just review this.

File: docs/en-US/v2.3.x/web/session/README.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+---
```
Comment (XdpCs) [inline docs/en-US/v2.3.x/web/session/README.md:1]: just review this.

File: docs/zh/v2.3.x/web/session/README.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
+---
```
Comment (XdpCs) [inline docs/zh/v2.3.x/web/session/README.md:1]: just  review this.

**Review by flycash — APPROVED** [review]

**Pull request merged**